data_transformation_plugins/ecco_darwin_transformation.py

```diff
@@ -0,0 +1,47 @@
+import xarray
+import re
+
+def ecco_darwin_transformation(file_obj, name, nodata):
+    """Tranformation function for the ecco darwin dataset
+
+    Args:
+        file_obj (s3fs object): s3fs sile object for one file of the dataset
+        name (str): name of the file to be transformed
+        nodata (int): Nodata value as specified by the data provider
+
+    Returns:
+        dict: Dictionary with the COG name and its corresponding data array.
+    """
+    var_data_netcdf = {}
+    xds = xarray.open_dataset(file_obj)
+    xds = xds.rename({"y": "latitude", "x": "longitude"})
+    xds = xds.assign_coords(longitude=((xds.longitude / 1440) * 360) - 180).sortby(
+        "longitude"
+    )
+    xds = xds.assign_coords(latitude=((xds.latitude / 721) * 180) - 90).sortby(
+        "latitude"
+    )
+
+    variable = [var for var in xds.data_vars]
+
+    for _ in xds.time.values:
+        for var in variable[2:]:
```

Comment:
To make this code more readable replace this `variable = [var for var in xds.data_vars]`  by 
variables = list(xds.data_vars)[2:]
Then you can do something like
```python
for var in variables:
```